GitHome13/Salesforce/Pages/LoginPage.cs

```diff
@@ -0,0 +1,50 @@
+﻿using GitHome13.Core;
+using GitHome13.Core.Elements;
+using GitHome13.Helpers;
+using GitHome13.Pages;
+using OpenQA.Selenium;
+using SeleniumTests.SwagLabs;
+
+namespace GitHome13.Salesforce.Pages
+{
+    internal class LoginPage : BasePage
+    {
+        private Input userNameInput = new(By.XPath("//input[@name='username']"));
+        private Input passwordInput = new(By.XPath("//input[@name='pw']"));
+        private Input loginButton = new(By.XPath("//input[@name='Login']"));
+
+        public const string url = "https://p1e-dev-ed.develop.my.salesforce.com/";
+
+        public LoginPage()
+        {
+
+        }
+
+        public NewAccountModal Login()
+        {
+            new LoginPage().OpenPage().TryToLoginByUserModel(UserBuilder.GetSalesforceUser());
+
+            Browser.Instance.NavigateToUrl("https://p1e-dev-ed.develop.lightning.force.com/lightning/o/Account/list?filterName=Recent");
```

Comment:
instead off direct navigation create  more user behavior approach, navigate to the page via elements clicking